frontend/app/templates/components/home-menu.hbs

```diff
@@ -0,0 +1,22 @@
+<nav class="navbar navbar-default">
+  <div class="container-fluid">
+    <div class="navbar-header">
+      <button type="button" class="navbar-toggle collapsed"
+      data-toggle="collapse" data-target="#menu-collapse">
+        <span class="sr-only">Toggle navigation</span>
+        <span class="icon-bar"></span>
+        <span class="icon-bar"></span>
+        <span class="icon-bar"></span>
+      </button>
+      {{ link-to 'Home' 'index' class="navbar-brand" }}
```

Comment:
Ummm sorry ... I mis-guided you on that ... The inline version of link-to is not yet _officially deprecated_. But there are talks to deprecate it soon. (For example. https://github.com/emberjs/ember.js/pull/13438#issuecomment-216200074). 

And also, see the advantages of the block form here https://github.com/rwjblue/ember-template-lint/blob/master/docs/rule/inline-link-to.md .

So, its best we stick to the block form through out. 

--- 

As for your logout, create a route called as logout. And within that route's `beforeModel` method, make an API request to `/logout` and then do `this.replaceWith('index')`. 

Now, for your logout button, use link-to and make it goto the logout route, and the user will be logged out.